apps/i18n/common/en_us.json

```diff
@@ -68,7 +68,15 @@
   "advancedSettings": "Advanced Settings",
   "age": "Age",
   "agenda": "Agenda",
+  "aiAssessment": "AI Assessment",
+  "aiAssessmentDoesMeet": "Extensive or Convincing Evidence",
+  "aiAssessmentDoesNotMeet": "Limited or No Evidence",
   "aiAssessmentEnabled": "AI Assessment Enabled",
+  "aiAssessmentExplanation": "PLACEHOLDER TEXT FOR INFO TIP",
```

Comment:
nit: don't add text that shouldn't be translated to this file

Reply:
Yep. Translators see the strings in this file and its more trouble than it is worth to go hide them. Easy fix: just put a string like this as a constant directly in the module it is used.